src/app/error.tsx

```
@@ -0,0 +1,25 @@
+'use client';
+
+import { Button } from '@/components/ui/button';
+
+export default function Error({
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)